README.md

```diff
@@ -82,9 +82,11 @@ exceptions though (like photos).
 
 ### Linking to Posts
 To link other posts, please use the following local URL syntax instead of a global URL:
+Example : Global URL : http://0.0.0.0:4000/introducing-kartothek/
```

Comment:
Please convert this on into a list. Your current style is somewhat "invalid" markdown (new paragraphs require empty lines), see:

![Screenshot 2020-08-03 at 09 38 00](https://user-images.githubusercontent.com/57095103/89157660-2e1c9c80-d56d-11ea-8223-2d51da9d1113.png)

Reply:
done.